src/db/models/planTag.ts

```diff
@@ -5,6 +5,11 @@ import type { Brand } from '../../util/types';
 import { defineIDModel } from '../util/id-model';
 import { PLAN_ID } from './plan';
 
+const PLAN_PUBLISH_TYPE = t.keyof({
+  Major: null,
+  Minor: null,
+  Custom: null,
+});
```

Comment:
I don't think any other similar types are capitalized.
```suggestion
const PLAN_PUBLISH_TYPE = t.keyof({
  major: null,
  minor: null,
  custom: null,
});
```